### PR TITLE
Automatically create and keep up-to-date a list of wrapped ERC20 tokens

### LINKED ISSUE #455: automatically create and keep up-to-date a list of wrapped ERC20 tokens
Each wrapped ERC20 token has the same name and symbol, which creates a poor UX when interacting with them in dapps like Uiswap.

We can combat this by creating and maintaining a list of conditional tokens, which can then be imported into Uniswap to provide additional metadata for each token.

Here is an [example list](https://gist.github.com/auryn-macmillan/4aa431933bd86925891815208a9734d9).

Our list should be compatible with, and listed on, [token list](https://tokenlists.org/), here are the [instructions for authoring a list](https://github.com/Uniswap/token-lists#authoring-token-lists).

The list could perhaps be hosted in this repo or on IPFS.

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #455

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr792--conditionaltokensexplorer.review.gnosisdev.com

File: conditional_token_lists/conditional-token-list-chain-4.json [hunk 1]
```diff
@@ -0,0 +1,682 @@
+{
```
Comment (hpmaxi) [inline conditional_token_lists/conditional-token-list-chain-4.json:1]: Should we version this folder?

**Review by hpmaxi — APPROVED** [review]
This is working, I tried in uniswap adding the list and it works (Uploading another generated list with other hash)

Maybe there is room for improving the error handling in case of an invalid `infuraId`, or a network different than rinkeby or mainnet but truthy anyway. But I don't think that is necessary 

Captura de pantalla de 2021-01-06 12-32-25

File: conditional_token_lists/conditional-token-list-chain-4.json (hunk 1 above)
Comment (mariano-aguero) [reply conditional_token_lists/conditional-token-list-chain-4.json:1]: I think yes, for the moment, the requirement says `The list could perhaps be hosted in this repo or on IPFS.`

**Pull request merged**